cfgov/v1/models/base.py

```diff
@@ -549,19 +569,24 @@ def too_many_attempts(self, value, timestamp):
         attempts = self.failed_attempts.split(',')
         return len(attempts) > value
 
+
 class TemporaryLockout(models.Model):
     user = models.ForeignKey(User)
     created = models.DateTimeField(auto_now_add=True)
     expires_at = models.DateTimeField()
 
 
 class Feedback(models.Model):
-    is_helpful = models.BooleanField()
-    comment = models.TextField(blank=True, null=True)
+    submitted_on = models.DateTimeField(auto_now_add=True)
     page = models.ForeignKey(
         Page,
         related_name='feedback',
         null=True,
         on_delete=models.SET_NULL,
     )
-    submitted_on = models.DateTimeField(auto_now_add=True)
+    comment = models.TextField(blank=True, null=True)
+    referrer = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
referrer is intended for a stand-alone feedback page that could be linked to from any number of other pages. in those cases, the product owners want to know which page the user came from.